src/test/features/ccj/routes/paid-amount.ts

```diff
@@ -0,0 +1,94 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import '../../../routes/expectations'
+
+import { Paths } from 'ccj/paths'
+
+import { app } from '../../../../main/app'
+
+import * as idamServiceMock from '../../../http-mocks/idam'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
+import { checkAuthorizationGuards } from './checks/authorization-check'
+import { PaidAmountOption } from 'ccj/form/models/yesNoOption'
+import { sampleClaimObj } from '../../../http-mocks/claim-store'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const externalId = sampleClaimObj.externalId
+const paidAmountPage = Paths.paidAmountPage.uri.replace(':externalId', externalId)
+const paidAmountSummaryPage = Paths.paidAmountSummaryPage.uri.replace(':externalId', externalId)
+
+const validFormData = {
+  option: PaidAmountOption.YES.value,
+  amount: 10
+}
+
+describe('CCJ - paid amount page', () => {
+  attachDefaultHooks()
+
+  describe('on GET', () => {
+    checkAuthorizationGuards(app, 'get', paidAmountPage)
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta')
+      })
+
+      it('should return 500 and render error page when cannot retrieve claims', async () => {
```

Comment:
The same here, it would be good to see the behaviour when:

- retrieving a CCJ draft failed
- that page is not accessible when claim is not eligible for CCJ

Reply:
Added test for "retrieving a CCJ draft failed"